### ISSUE TITLE
v2.1.3 Installer installs 2.1.0?!

### ISSUE THREAD

**Issue by NealGooch** [GNS3/gns3-gui#2430]
Title says it all - I have re-downloaded several times and it does it every time!

**Comment by ziajka** [comment]
Thanks @NealGooch for reporting, indeed it shows 2.1.0 is installed. Next week we will release 2.1.4

**Comment by grossmj** [comment]
@NealGooch

Have you uninstalled version 2.1.0 before installing version 2.1.3? Did you
get a request to uninstall the previous version while installing 2.1.3?

**Comment by sairuscz** [comment]
Had the same issue on Windows 10 when upgrading from 2.1.0 to 2.0.3. The solution was to manually remove C:\Program Files\GNS3\ before running the new installer. I am sure a regular uninstall of the old version would work as well. I used GNS3-2.1.3-all-in-one.exe installer. 

What is the difference between GNS3-2.1.3-all-in-one-swi.exe and GNS3-2.1.3-all-in-one.exe anyway?

**Comment by NealGooch** [comment]
Ah - I had uninstalled the previous version before I installed 2.1.3 but I hadn't checked that the files in "C:\Program Files\GNS3" had been deleted - they hadn't.

So on trying yet again I deleted the app using the uninstall option then manually deleted all the contents  of the directory.  

Then I installed all again and this time everything is good.

What I had forgotten to say is getting the version number right is important as I use a remote server that I had already upgraded!

I'm now going to try uninstalling 2.1.3 to see if it does delete the files or not!
